test/powershell/engine/HelpSystem.OnlineHelp.Tests.ps1

```diff
@@ -0,0 +1,67 @@
+Describe 'Online help tests for PowerShell Core Cmdlets' -Tags "CI" {
```

Comment:
The tests don't test the process launch at all. Shall we at least have 1 test exercise the new code?